### PR TITLE
[1LP][RFR] Daily and Hourly graph tests

### PR DESCRIPTION
**Pull request by digitronik**
Purpose or Intent
=================
- Automating Hourly and Daily graph for Vsphere Provider

{{pytest: cfme/tests/candu/test_vm_graph.py::test_graph_screen -v --use-provider complete}}

### CODE REVIEW

File: cfme/fixtures/candu.py [hunk 1]
```diff
@@ -25,3 +25,16 @@ def enable_candu(appliance):
...
+def collect_data(appliance, provider, interval='hourly', back='7.days'):
+    """Collect hourly back data for vsphere provider"""
+    vm_name = provider.data['cap_and_util']['chargeback_vm']
+
+    # Capture real-time C&U data
+    ret = appliance.ssh_client.run_rails_command(
+        "\"vm = Vm.where(:ems_id => {}).where(:name => {})[0];\
+        vm.perf_capture({}, {}.ago.utc, Time.now.utc)\""
+        .format(provider.id, repr(vm_name), repr(interval), back))
+    yield ret.success
```
[the remark this answers is not in the document]
Comment (digitronik) [reply cfme/fixtures/candu.py]: Right, but its common fixture. Going to use in other placess where me and Nandini like to assert success message.

[the remark this answers is not in the document]
Comment (digitronik) [reply cfme/fixtures/candu.py]: @quarckster Yes, but as per my understanding, we have to go with `yield` for `generative` objects its good to use `return` in normal definition. As its pytest fixture and parametrize as per provider means generative better to go with yield. Please, let me know your intention way I should go with a return.

[the remark this answers is not in the document]
Comment (quarckster) [reply cfme/fixtures/candu.py]: `yield` only makes sense if a fixture has the teardown part. You fixture doesn't have it.

File: cfme/fixtures/candu.py [hunk 2]
```diff
@@ -25,3 +25,16 @@ def enable_candu(appliance):
...
+def collect_data(appliance, provider, interval='hourly', back='7.days'):
+    """Collect hourly back data for vsphere provider"""
+    vm_name = provider.data['cap_and_util']['chargeback_vm']
+
+    # Capture real-time C&U data
+    ret = appliance.ssh_client.run_rails_command(
+        "\"vm = Vm.where(:ems_id => {}).where(:name => {})[0];\
+        vm.perf_capture({}, {}.ago.utc, Time.now.utc)\""
+        .format(provider.id, repr(vm_name), repr(interval), back))
+    return ret.success
```
[the remark this answers is not in the document]
Comment (digitronik) [reply cfme/fixtures/candu.py]: Yes, This fixture is for data collection l.e. historical data with the rhel commands. I will use in other places where like to check actually the command executed successfully run or not.
`ret.success` return success message only.

[the remark this answers is not in the document]
Comment (quarckster) [reply cfme/fixtures/candu.py]: Yes, so I ask do you use `ret.success` anywhere? Is it really need to return that value?

[the remark this answers is not in the document]
Comment (digitronik) [reply cfme/fixtures/candu.py]: For now, it's not necessary but I like to go with possibilities; for eg https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/intelligence/chargeback/test_chargeback_long_term_rates.py#L156

**Pull request merged**